File: Python/010.py
def solve(n):
    sieve = [True] * (n + 1)
    s = 0
    for i in range(2, n + 1):
        if sieve[i] == True:
            s += i
            for j in range(i + i, n + 1, i):
                sieve[j] = False
    return s

# n = 10
n = 2000000
print(solve(n))

File: C++/010.cpp
#include <iostream>
#include <vector>
using namespace std;

typedef unsigned long long LongInt;

LongInt solve(LongInt n)
{
    vector<bool> sieve(n + 1, true);
    LongInt sum = 0;
    for (int i = 2; i <= n; i++)
        if (sieve[i] == true) {
            sum += i;
            for (int j = i + i; j <= n; j += i) 
                sieve[j] = false;
        }
    return sum;
}

int main()
{
    // int n = 10;
    int n = 2000000;
    cout << solve(n) << endl;
}